src/style.css

```
@@ -0,0 +1,93 @@
+:root {
+  --shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  --color: #3d4e61;
+}
+@font-face {
+  font-family: Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.label_form {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: center;
+}
+
+.label {
+  margin: 20px 0;
+}
+
+.icons {
+  position: absolute;
+  background-image: url(/images/Search.svg);
+  z-index: 2;
+  background-size: cover;
+  background-position: center;
```

Combine with `background` property
```suggestion
  z-index: 2;
  
  background: .....your values;
```